NETDefault/Gateway.Buckaroo/Mappers/BuckarooRequestMapper.cs

```diff
@@ -0,0 +1,239 @@
+﻿using Gateway.Buckaroo.Models.Dto.Requests;
+using Gateway.Buckaroo.Models.Request;
+
+namespace Gateway.Buckaroo.Mappers;
+
+internal static class BuckarooRequestMapper
+{
+    internal static GetDebtorInfoRequestDto Map(GetDebtorRequest request)
+        => new()
+        {
+            Services = new ServiceDto
+            {
+                ServiceList = new List<ServiceListDto>
+                {
+                        new ServiceListDto
+                        {
+                            Name = "CreditManagement3",

```

Comment:
I see a lot of these strings in this class and in other classes. Perhaps put them in a `Constants` and include that using a `using static [..].Constants`, so you can use its properties directly (like `CreditManagement` for example)?